lib/generators/hyrax/work/work_generator.rb

```diff
@@ -1,70 +1,41 @@
 require 'rails/generators'
 
-class Rails::Generators::NamedBase
-  private
-
-    def destroy(what, *args)
-      log :destroy, what
-      argument = args.map(&:to_s).flatten.join(' ')
-
-      in_root do
-        run_ruby_script("bin/rails destroy #{what} #{argument}", verbose: true)
-      end
-    end
-end
-
 class Hyrax::WorkGenerator < Rails::Generators::NamedBase
   source_root File.expand_path('../templates', __FILE__)
 
   argument :attributes, type: :array, default: [], banner: 'field:type field:type'
 
   # Why all of these antics with defining individual methods?
   # Because I want the output of Hyrax::WorkGenerator to include all the processed files.
-  def create_model_spec
```

Comment:
I reordered the generator methods to generate its files in alphabetical order (app, then config, then spec).